nuitka/build/static_src/MetaPathBasedLoader.c

```diff
@@ -34,6 +34,12 @@
 #endif
 
 #include "nuitka/prelude.h"
+// Include definition of PyInterpreterState, hidden since Python 3.8
+#if PYTHON_VERSION > 380
```

Comment:
This should probably be >= 380, otherwise it ought to not have an effect before 3.8.1

Reply:
Yes, correct. I needed to change it. Forgot to repush.

Reply:
Ok, please do so then :-)

Reply:
Got back from a mini-holiday and re-pushed as is with this bit fixed.
Happy to move it `prelude.h` if you think it is best now that you've played around with py3.8 a bit more.

I see the tests at https://github.com/Nuitka/Nuitka-CPython-tests/tree/CPython38 , but with timestamps of Jun 19, 2017.

Reply:
Ah, and moving it, no, you convinced me. I will merge once I hit standalone mode, or whatever triggers the need for me, right now however, I was not working on this, my private issues continue.